src/Sylius/Behat/Context/Ui/Admin/Helper/NavigationTrait.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Context\Ui\Admin\Helper;
+
+trait NavigationTrait
+{
+    public function hasShowPageButton(): bool
+    {
+        return $this->hasElement('show_' . $this->getResourceName() . '_button');
```

Comment:
If anything, we should make two methods `getXButtonElement` for instance, instead of creating the element name on the go. Or if extracted to an element, just using  `edit_button`/`show_button` should be fine.